guardrails/llm_providers.py

```diff
@@ -551,7 +551,7 @@ def _invoke_llm(self, *args, **kwargs) -> LLMResponse:
         # Get the response from the callable
         # The LLM response should either be a
         # string or an generator object of strings
-        llm_response = self.llm_api(*args, **kwargs)
+        llm_response = self.llm_api(*args, **kwargs)  # type: ignore
```

Comment:
@CalebCourier not sure  `#type: ignore`  this is the way this should be handled now None is possible but we both know its not possible but the static analysis doesnt? thoughts?

Reply:
@dtam if it can never happen in practice/reality, then this is a situation where we'd use an ignore.  I think we should also start adding `TODO`'s to see if there is a better way to type it where we don't have to use as many ignores, but there are cases where the static type analysis is just wrong.